custom/icds/management/commands/export_orphaned_cases.py

```diff
@@ -0,0 +1,65 @@
+import csv
+
+from django.core.management import BaseCommand
+
+from corehq.apps.es import CaseES
+from corehq.apps.es import filters
+
+from dimagi.utils.chunked import chunked
+
+from corehq.util.log import with_progress_bar
+
+CHILD_PROPERTIES = ['case_id', 'owner_id', 'opened_on', 'server_modified_on',
+                    'name', 'indices', 'aadhar_number', 'dob', 'died']
+
+
+class Command(BaseCommand):
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            'hh_file',
+            help="File path for household case output file",
+        )
+        parser.add_argument(
+            'child_file',
+            help="File path child case output file",
+        )
+
+    def handle(self, hh_file, child_file, **options):
+        hh_cases = self._get_closed_hh_cases()
+        with open(hh_file, 'w') as hh_csv, open(child_file, 'w') as child_csv:
+            hh_writer = csv.writer(hh_csv)
+            child_writer = csv.writer(child_csv)
+            for cases in chunked(with_progress_bar(hh_cases), 500):
```

Comment:
This will fail because `with_progress_bar` need to know the total. Options are to pass in the total `hh_cases.count` or implement `__length__` on `ScanResult`